modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -3,88 +3,614 @@
+
+Also, the `editimage_proxy` option has been renamed to `export_image_proxy_service_url`. The option was renamed to make it easier to configure when using https://tiny.cloud/docs/enterprise/server/[Tiny services].
+
+Finally, `editimage` also offers a new export plugin option: `export_image_proxy_service_url`.
+
+NOTE: `editimage_proxy` (the renamed equivalent to the previous `imagetools_proxy`) is still available as an alternative configuration option for setting up image proxy functionality without using Tiny services.
+
+Users of the Free version of {productname} 5.x who rely on the, now deprecated and removed, `imagetools` plugin, should not upgrade to {productname} 6.0 without a https://tiny.cloud/pricing/[subscription plan] that provides access to Premium plugins such as https://tiny.cloud/tinymce/features/#productivity[_EditImage_].
+
+NOTE: No Premium Plugins released with {productname} 6.0 are backwards compatible with earlier versions of {productname}.
+
+=== Media
+
+As noted in the xref:6.0-release-notes.adoc#changed-plugins-media[_{productname} 6.0 Release Notes_], the `media` plugin no longer uses the SaxParser.
+
+As a consequence the `iframe`, `video`, `audio`, and `object` elements are no longer marked as _special_, and cannot be marked as such. Put another way, the DomParser does not preserve embedded content.
+
+Instead these elements, and their contents, are validated against the {productname} schema.
+
+This validation is generally done by the `valid_elements`, `valid_children` or `extended_valid_elements` config.
+
+The schema must, therefore, be configured to allow usage of the media elements, and the media element attributes.
+
+NOTE: As of this release, there are no known cases of this change causing media objects to not work as expected. But, also as of this release, not every potential or possible use-case has been tested.
+
+[[plugins-paste]]
+=== Paste
+
+As noted in the xref:6.0-release-notes.adoc#removed-or-deprecated-plugins-paste[_{productname} 6.0 Release Notes_], the `paste` plugin’s functionality is no longer provided as a plugin. It is now a core part of {productname} 6.0.
+
+In conjunction with this change, three changes have been made:
+
+. A `paste` default has changed. The `paste_data_images` option now defaults to `true`.
++
+When `paste` was a plugin, this option was, by default, set to `false`, which prevented images being pasted from the local machine.
++
+. The `mceInsertClipboardContent` argument, `content`, has been renamed. It is now `html`.
++
+The new name is a more accurate reflection of what sort of data the argument passes.
++
+The `content` argument can no longer be used with `mceInsertClipboardContent`. If `content` is used, no data is passed. `mceInsertClipboardContent` only attempts to pass data if the `html` or `text` property is used.
++
+NOTE: `PastePreProcess` also takes an argument, `content`. This argument, for this command, has not been changed.
```

Comment:
As noted in the plugins page this isn't correct and I think the best course of action would be to just remove it as it's always been passed a `content` property.
```suggestion
```